src/common/http/filename_type.cpp

```diff
@@ -0,0 +1,24 @@
+#include "filename_type.hpp"
+#include "../gcode_filename.hpp"
```

Comment:
Would it be better to just unify everything under one (renamed) file instead of having one half in the gcode_filename and another in filename_type? The code is kind of similar to each other…

Reply:
My thoughts were, that the functions `file_type_by_ext` and `file_type` are specific to networking (the return value in this form does not seem to me to be likely used anywhere else), so I though I would put it inside `common/http`, but the gcode function is usable (and used already) in different places in the FW, :thinking:  But by this logic the `filename_is_firmware` should probly be also in `common` because that one might be used somewhere else.

Or we can just put it all in `src/common/filename_type.(cpp|hpp)` :man_shrugging:

Reply:
Yes, something like that (`src/common/filename_type`) was what I've had in mind.